UserManagement.BL/UserManager.cs

```diff
@@ -31,6 +38,57 @@ public List<string> GetAllUserNames()
             return repository.GetUsers().Select(u => u.UserName).ToList();
         }
 
+        /// <summary>
+        /// Creates Login record
+        /// </summary>
+        /// <param name="user"></param>
+        public bool CreateLoginRecord(User user)
+        {
+            if (user == null)
+            {
+                return false;
+            }
+
+            var login = new LoginLog()
```

Comment:
`LoginAttempt`? `Log` in all other contexts means some data that is used for figuring out what's wrong. However, here, in my opinion, it can be used to forbid multiple login attempts from the same IP.